tests/integration_tests/functional/test_mmds.py

```diff
@@ -122,6 +122,70 @@ def test_custom_ipv4_support_config(test_microvm_with_ssh, network_config):
     )
 
 
+def test_json_response(test_microvm_with_ssh, network_config):
+    """Test the API for MMDS custom ipv4 support."""
+    test_microvm = test_microvm_with_ssh
+    test_microvm.spawn()
+
+    response = test_microvm.mmds.get()
+    assert test_microvm.api_session.is_status_ok(response.status_code)
+    assert response.json() == {}
+
+    data_store = {
+        'latest': {
+            'meta-data': {
+                'ami-id': 'ami-12345678',
+                'reservation-id': 'r-fea54097',
+                'local-hostname': 'ip-10-251-50-12.ec2.internal',
+                'public-hostname': 'ec2-203-0-113-25.compute-1.amazonaws.com',
```

Comment:
Is this a valid json? Are `,` accepted even though there is no other field afterwards?

Reply:
This is a python dict, so it doesn't really matter. Removed it since it is not needed.